tests/cephfs/cephfs_system/cephfs_systest_sv_clone_ops.py

```diff
@@ -0,0 +1,395 @@
+import datetime
+import json
+import logging
+import os
+import random
+import threading
+import time
+import traceback
+from multiprocessing import Value
+from threading import Thread
+
+from ceph.parallel import parallel
+
+# from ceph.ceph import CommandFailed
+from tests.cephfs.cephfs_system.cephfs_system_utils import CephFSSystemUtils
+from utility.log import Log
+
+log = Log(__name__)
+logger = logging.getLogger("run_log")
+
+logging_thread = None
+mds_logging_thread = None
+stop_event = threading.Event()
+
+
+def run(ceph_cluster, **kw):
+    """
+    System test - SV and Clone ops : Run below workflows in parallel and repeat until runtime,
+
+    1. Create 100 subvolumes. Delete the subvolumes and in parallel create another 100 subvolumes.
```

Comment:
It is good to have these configurations made custom variables and pass to the test. If next time, scaling is needed, we could increase the count without modifying the code.

Reply:
It is customizable neha, please check code below at line40,
        sv_cnt = config.get("sv_cnt", 300)
        clone_cnt = config.get("clone_cnt", 10)